src/main/java/duke/exceptions/EmptyCommandException.java

```diff
@@ -0,0 +1,16 @@
+package duke.exceptions;
+
+/**
+ * EmptyCommandException represents an exception if
+ * the existing command is empty.
+ */
+public class EmptyCommandException extends Exception {
```

Comment:
I like that the names of your exceptions are clear and appropriate for each error.